__test__/useCases/GetCommitmentsAndExportFile/index.e2e.test.ts

```diff
@@ -0,0 +1,43 @@
+import axios from "axios";
```

Comment:
Instead of creating the folder `useCases` we could only have the name of the thing we are testing, like `Commitment`. What do you think?